cypress/plugins/getCypressEnvVariables.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
```

Comment:
@JoakimSM what we use this file for?

Reply:
Maybe write a comment about this file as well @JoakimSM (?)

Reply:
Absolutely 👍